index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <link rel="stylesheet" href="main.css">
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+</head>
+
+<body>
+ <header>
+  <div class="header-left">
+   <h2>CodePen Home
+    FCC: Tribute Page</h2>
+  </div>
+
+  <div class="header-right">
+   <ul>
+    <li><a>Love</a></li>
+    <li><a>Change View</a></li>
+    <li><a>Sign Up</a></li>
+    <li><a>Log in</a></li>
+   </ul>
+  </div>
+ </header>
+ <div id="main">
+  <h1 id="title">Dr. Norman Borlaug</h1>
+  <h4 id="tribute-info">The man who saved a billion lives</h4>
+  <div id="img-div">
+   <img id="image" src="./image/img.jpg" alt="picture of Dr.Normal Borlaug">
+   <div id="img-caption"> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
+    yields - part of his life-long war on hunger.</div>
+  </div>
+
+  <div class="timeline-div">
```

Comment:
I would personally use less div's and more semantic tags like <section> or <article>

Reply:
As in the previous comments, remember to explain the reasons for asking for a change.

Reply:
I would personally use less div's and more semantic tags like section or article.
In real project you will have a lot of benefits using semantic tags, like
Improving the meaning of the page

Reply:
I would personally use less div's and more semantic tags like

or
Semantic tags gives gives you lots of benefits,here are few of them:
Gives more meaning to the web browser and screen readers
Helps with search engines
Easier debugging and understanding